src/main/services/GuildService.java

```diff
@@ -630,23 +646,21 @@ public static String getAccountIDUser()
 	}
 	public static boolean checkPermission(Connection connection, String permission) throws SQLException, TokenException, NotFoundException {
 		try {
-			boolean  check= false;
+			boolean  check = false;
 			List<String> listGuild =  Guild.getAllGuildByAccountId(connection,getAccountIDUser());
 			for (String guild : listGuild) {
-				List<GuildPermission> permissionList = GuildPermission.getAllByAccountIdAndGuildId(connection,getAccountIDUser(),Guild.getIdByName(connection,guild));
-				for (GuildPermission guildPermission : permissionList) {
-					if (guildPermission.getName().equals(permission)){
-						check = true;
-					}
+				check = AuthService.GuildAuthorization(connection,getAccountIDUser(),Guild.getIdByName(connection,guild),permission);
+				if (check) {
+					return check;
```

Comment:
How can I unsure that I have permission in current guild ? 
I see u check permission but get all guild, when a role in a guild have that permission, it will be passed!
Check again, pls!